analyzer/src/main/kotlin/managers/NPM.kt

```diff
@@ -129,12 +129,12 @@ open class NPM(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig:
     override fun getVersionRequirement(): Requirement = Requirement.buildNPM("5.7.* - 6.4.*")
 
     override fun mapDefinitionFiles(definitionFiles: List<File>) =
-            PackageJsonUtils.mapDefinitionFilesForNpm(definitionFiles).toList()
+        PackageJsonUtils.mapDefinitionFilesForNpm(definitionFiles).toList()
 
     override fun prepareResolution(definitionFiles: List<File>) =
-            // We do not actually depend on any features specific to an NPM version, but we still want to stick to a
-            // fixed minor version to be sure to get consistent results.
-            checkVersion(ignoreActualVersion = analyzerConfig.ignoreToolVersions)
+    // We do not actually depend on any features specific to an NPM version, but we still want to stick to a
```

Comment:
just marking another type of indent which seems a bit surprising, haven't read the official style  guide ...

Reply:
Me neither 😀 I just had the IDE reformat the code as described at https://kotlinlang.org/docs/reference/code-style-migration-guide.html#migration-to-a-new-code-style.